hub/powertoys/color-picker.md

```diff
@@ -9,31 +9,31 @@ no-loc: [PowerToys, Windows, ColorPicker, Color, Picker]
 
 # Color Picker utility
 
-A system-wide color picking utility for Windows to pick colors from any screen and copy it in a configurable format to your clipboard.
+A system-wide color picking utility for Windows to pick colors from any screen and copy it in a configurable format to the clipboard.
 
 ![ColorPicker.](../images/pt-colorpicker-hex-editor.png)
 
 ## Getting started
 
 ### Enabling Color Picker
 
-To start using Color Picker, enable it in the PowerToys Settings (**Color Picker** tab).
+Enable Color Picker in the **Color Picker** tab in PowerToys.
 
 ### Activating Color Picker
 
-Once enabled, you can choose one of the following three behaviors to be executed when opening Color Picker with the activation shortcut (default: <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>C</kbd>).
+You can choose what happens when you activate Color Picker (default: <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>C</kbd>) by changing **Activation Behavior**:
 
 :::image type="content" source="../images/pt-colorpicker-activation.gif" alt-text="ColorPicker behaviors.":::
 
-- **Open editor** - Opens editor window directly, from here you can choose a color from the history, fine-tune a selected color, or capture a new color by opening the color picker.
-- **Pick a color and open editor** - Opens Color Picker. After selecting a color, the editor is opened and the selected color is copied to the clipboard (in the default format - configurable in the Settings dialog).
-- **Only pick a color** - Opens Color Picker only and the selected color will be copied to the clipboard.
+- **Open editor** opens an editor that lets you choose a color from the colors history, fine-tune a selected color, or pick a new color
+- **Pick a color and open editor** activates Color Picker, then opens an editor and copies the selected color to the clipboard after you've picked a color
+- **Only pick a color** activates Color Picker and copies the selected color to the clipboard
 
-### Selecting colors
+### Picking colors
 
-After the Color Picker is activated, hover your mouse cursor over the color you would like to copy and left-click the mouse button to select a color. If you want to see the area under your cursor in more detail, scroll up to zoom in.
+After activating Color Picker, hover over a color and click to pick that color. If you want to see the area under your cursor in more detail, scroll up to zoom in.
```

Comment:
This is an improvement over the original sentence, but it would be better to avoid the work "click". I may be on a touch device, tapping the controls. That's why we typically use "select" instead of "click".

Reply:
Agreed! I've committed a change for this now. Let me know what you think.